README.md

```diff
@@ -1 +1,63 @@
-# PokemonAPI
\ No newline at end of file
+# Pokémon API
+> This web app displays different Pokémon gotten from the RESTful [Pokémon API](https://pokeapi.co/) in which you can like your favorites. It also allows you to see further information about each Pokémon and leave a comment. All data is recorded thanks to the [Involvement API service](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270). Built with Javascript.
+
+## 🛠️ Built With
+
+- HTML, CSS & Javascript
+- Webpack
+- Jest
+- Babel
+- Linters: Lighthouse, Webhint, Stylelint & ESlint
+
+## 📹 [Video documentation](https://www.loom.com/share/febaae743e004d9aa35818b252fdca95)
+
```

Comment:
- [ ] Kindly add a live link to the website.

Reply:
Hello, @ahmetbozaci Thanks for the review 😁 

Right now when you click on the video documentation link it opens the loom video
Could you explain this point in more detail? What else should I add to it?

Reply:
- Yes kindly deploy and add the live link.Video link is working.

Reply:
@ahmetbozaci Thanks for clarifying it. Live link added. 👌🏼